resources/models.py

```diff
@@ -2,30 +2,34 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship
 
+
 Base = declarative_base()
 
+
 class Department(Base):
     __tablename__ = 'Departments'
     ID_department = Column(Integer, primary_key=True)
     Department_name = Column(String)
     Description = Column(String)
     Contact_info = Column(String)
 
+
```

Comment:
did you use any code formatter? or just add it manually

Reply:
It's from PEP 8 style guide: https://peps.python.org/pep-0008/#blank-lines.
>"Surround top-level function and class definitions with two blank lines."